tv_apps/app-deck/build/build.js

```diff
@@ -4,10 +4,25 @@
 
 var utils = require('utils');
 
+var AppDeckBuilder = function() {};
+
+// XXX: we should extract the logic of copying default-cards.json
+// to one common place. See https://bugzil.la/1123585
+AppDeckBuilder.prototype.execute = function(options) {
+  var stageDir = utils.getFile(options.STAGE_APP_DIR);
+  var configFile = utils.getFile(stageDir.path, 'js', 'init.json');
+  var defaultConfigFile = utils.getFile(options.GAIA_DIR, 'tv_apps',
+                                'tv_shared', 'resources', 'default-cards.json');
+  var defaultConfig = utils.getJSON(defaultConfigFile);
```

Comment:
If we do not support customization, we may just copy the file `default-cards.json` to `.../js/default-cards.json`. It's not hard to change the code in TVSharedUtils to add the copy of json files. We should do it and put `link tag` to html file, like:
`<link rel="resource" type="application/json" href="/shared/resources/tz.json"/>`